packages/experiments/src/components/Chiclet/ChicletCard.styles.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  normalize
+} from '../../Styling';
+import { IChicletCardStyleProps, IChicletCardStyles } from './ChicletCard.types';
+
+export const getStyles = (
+  props: IChicletCardStyleProps
+): IChicletCardStyles => {
+  const { theme } = props;
+  const { palette } = theme;
+
+  return ({
+    root: [
+      'ms-Chiclet',
+      normalize,
+      {
+        WebkitFontSmoothing: 'antialiased',
+        backgroundColor: palette.white,
+        borderRadius: '2px',
+        boxShadow: '0 1px 3px 0 rgba(0, 0, 0, 0.3)',
+        width: '600px',
```

Comment:
numerical pixel values in styling can be represented as numbers rather than strings. :)